calicoctl/calicoctl.go

```diff
@@ -89,7 +89,7 @@ Description:
 		case "ipam":
 			commands.IPAM(args)
 		case "config":
-			commands.Config(args)
+			//commands.Config(args)
```

Comment:
per node config is not yet supported in libcalico V2 so code for that is commented out

Reply:
okay